sympy/polys/ring_series.py

```diff
@@ -1898,8 +1898,9 @@ def _rs_series(expr, series_rs, a, prec):
     elif expr.is_Mul:
         n = len(args)
         for arg in args:    # XXX Looks redundant
-            R1, _ = sring(arg, expand=False, series=True)
-            R = R.compose(R1)
+            if not arg.is_Number:
```

Comment:
Not familiar with this routine, but could arg be `sqrt(2)`? If so then you should use `arg.is_number`.